test/test.js

```diff
@@ -376,7 +392,7 @@ versions.forEach((version) => describe(`Chunk implementation for minecraft ${ver
           }
         }
 
-        if (!version.startsWith('1.8')) {
+        if (!version.startsWith('1.8') && !chunkDump.includes('hypixel')) {
```

Comment:
which are hypixel chunks not the same after dumping?

Reply:
Because hypixel provides excess data which is dropped by pchunk (per this PR). See my comment here https://github.com/PrismarineJS/prismarine-chunk/pull/232#discussion_r1306124496

Since the test already iterates over all blocks and palettes in the chunk, it means the two chunks are still functionally the same if the test passes